src/loaders/VolumeLoaderUtils.ts

```diff
@@ -130,8 +141,33 @@ export function scaleMultipleDimsToSubregion(subregion: Box3, dims: ZYX[]): ZYX[
  */
 export function pickLevelToLoadUnscaled(loadSpec: LoadSpec, spatialDimsZYX: ZYX[]): number {
   const optimalLevel = estimateLevelForAtlas(spatialDimsZYX, loadSpec.maxAtlasEdge);
-  const levelToLoad = Math.max(optimalLevel + (loadSpec.scaleLevelBias ?? 0), loadSpec.multiscaleLevel ?? 0);
-  return Math.max(0, Math.min(spatialDimsZYX.length - 1, levelToLoad));
+  let levelToLoad = optimalLevel;
```

Comment:
Looks like there's not really a need for `optimalLevel` now that `levelToLoad` is immediately set to its value

Reply:
Oops, holdover from when I was also reporting optimalLevel!